.github/workflows/update-dataset-listings.yml

```diff
@@ -8,21 +8,25 @@ on:
 
 jobs:
   update-dataset-listings:
+    if: github.repository == 'nextstrain/nextstrain.org'
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-node@v1
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
       with:
-        node-version: 14
+        node-version: '14'
     - uses: nextstrain/.github/actions/setup-nextstrain-cli@master
+    # configure AWS necessary for `nextstrain remote upload`
+    - uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    - run: aws sts get-caller-identity
     - name: update dataset listings # currently just /staging and /influenza use these
       run: |
         npm ci
```

Comment:
Move `npm ci` up before `configure-aws-creds` here too?

Reply:
Oh right, I didn't read your first comment closely enough and I see this was suggested. Done.